src/couch/src/couch_bt_engine.erl

```diff
@@ -707,6 +814,23 @@ init_state(FilePath, Fd, Header0, Options) ->
             {compression, Compression}
         ]),
 
+    PurgeTreeState = case couch_bt_engine_header:purge_tree_state(Header) of
+        0 -> nil;
+        PTS -> PTS
+    end,
```

Comment:
Is this header upgrade logic? Should that be moved to `upgrade_purge_info/2` for clarity?